src/styles/common/_landing.scss

```diff
@@ -0,0 +1,118 @@
+.hero__container {
+  padding: 2%;
+  overflow-x: hidden;
+  width: 80%;
+}
+
+.hero__content {
+  background-color: #ffffff !important;
+  min-width: 80%;
+  display: block;
+}
+
+.hero__content--mobile {
+  display: none;
+  background-color: #ffffff !important;
+}
+
+.hero__text {
+  padding: 6%;
+  margin-left: 2rem;
+}
+
+.hero__section > * {
+  margin-bottom: 1rem;
+}
+
+.hero__section > p {
+  margin-bottom: 0.2rem;
+  font-size: 18px;
+}
+
+.hero__section span {
+  color: $secondary_color;
+}
+
+.hero__form {
+  margin-top: 2rem;
+}
+
+.input__section {
+  text-align: center;
+}
+
+.error__text {
+  color: $error_text_color;
+  height: 1rem;
+}
+
+.otp__button {
+  margin-top: 1.5rem !important;
+}
+
+.hero___imagesection {
+  position: relative;
+}
+
+.hero__image {
+  height: 500px;
+  object-fit: contain;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  margin-right: 2rem;
+}
+
+.hero__image--mobile {
+  height: 350px;
+}
+
+/* ------------ Media Queries ------------- */
+
+@media screen and (max-width: 985px) {
+  .hero__text {
+    padding: 6%;
+  }
+  .hero__image {
+    height: 450px;
+    margin-right: 0.5rem;
+  }
+  .hero__text {
+    margin-left: 0.5rem;
+  }
+  .hero__section h1 {
+    font-size: 2rem;
+  }
+  .hero__section > p {
+    margin-bottom: 0.2rem;
+    font-size: 1rem;
+  }
+}
+
+@media screen and (max-width: 830px) {
+  .hero__image {
+    height: 400px;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .hero__content {
+    display: none !important;
+  }
+  .hero__content--mobile {
+    display: block !important;
```

Comment:
Why do we need an !important here? we are overriding our own style in a media query, this should work without an !Important right?

Reply:
This className is given to a Grid component used from the carbon-components library which has the default set to block, I will look into this